viadot/sources/spectrum.py

```diff
@@ -0,0 +1,166 @@
+from typing import List, Dict, Literal
+import boto3
+
+import pandas as pd
+import awswrangler as wr
+
+
+class Spectrum:
+    """
+    A class for pulling data from and uploading to Spectrum.
```

Comment:
Spectrum -> AWS Redshift Spectrum. BTW, pls also change the file name to `redshift_spectrum`.

Reply:
Updated as per:
- [🎨 modified class name to AWSRedshiftSpectrum](https://github.com/dyvenia/viadot/pull/579/commits/f8e782320538289b4aa933de61917fc65933839e)
- [Rename spectrum.py to redshift_spectrum.py](https://github.com/dyvenia/viadot/pull/579/commits/438051b61cc0ecfc40f728f1f24188e4eb9ecd68)